protoc-gen-connect-kotlin/src/main/kotlin/com/connectrpc/protocgen/connect/internal/Parameters.kt

```diff
@@ -42,7 +42,8 @@ internal fun parse(input: String): Configuration {
     val parameters = parseGeneratorParameter(input)
     return Configuration(
         generateCallbackMethods = parameters[CALLBACK_SIGNATURE]?.toBoolean() ?: false,
-        generateCoroutineMethods = parameters[COROUTINE_SIGNATURE]?.toBoolean() ?: true, // Defaulted to true.
+        // Defaulted to true.
+        generateCoroutineMethods = parameters[COROUTINE_SIGNATURE]?.toBoolean() ?: true,
```

Comment:
IMO it was clearer before, but if this violates a lint rule it's not a big deal

Reply:
Yeah this failed a new lint rule that required comments to be on their own lines.